server/lib/queries.js

```diff
@@ -1153,6 +1154,49 @@ const getTransactionsTimeSeries = async (
   );
 };
 
+/**
+ * Returns tags along with their frequency of use.
+ */
+const getTagFrequencies = async args => {
```

Comment:
I don't understand why we have this change here, merging issue?

Reply:
cause the rebase I did from main